### PR TITLE
Closes #22

### LINKED ISSUE #22: Create dataset loader for GENIA Term Corpus
From http://www.geniaproject.org/genia-corpus/term-corpus

### PR DESCRIPTION
**Pull request by albertvillanova**
- **Name:** *GENIA Term corpus*
- **Description:** *The GENIA technical term annotation covers the identification of  physical biological entities as well as other important terms.*
- **Paper:** 
  - https://dl.acm.org/doi/abs/10.5555/1289189.1289260
  - https://www.semanticscholar.org/paper/GENIA-corpus-a-semantically-annotated-corpus-for-Kim-Ohta/da6c3fdf8ef9aae979a5dd156e074ba6691b2e2c
  - https://dl.acm.org/doi/10.5555/1567594.1567610
- **Data:** *http://www.geniaproject.org/genia-corpus/term-corpus*

Close #22.

### Checkbox

- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `biodatasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_BIGBIO_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `BigBioConfig` for the source schema and one for a bigbio schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_bigbio biodatasets/my_dataset/my_dataset.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: biodatasets/genia_term_corpus/genia_term_corpus.py [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+_DESCRIPTION = """\
+The identification of linguistic expressions referring to entities of interest in molecular biology such as proteins,
+genes and cells is a fundamental task in biomolecular text mining. The GENIA technical term annotation covers the
+identification of  physical biological entities as well as other important terms. The corpus annotation covers the full
+1,999 abstracts of the primary GENIA corpus.
+"""
+
+_HOMEPAGE = "http://www.geniaproject.org/genia-corpus/term-corpus"
+
+_LICENSE = """GENIA Project License for Annotated Corpora
```
Comment (jason-fries) [inline biodatasets/genia_term_corpus/genia_term_corpus.py:84]: Let's just use "GENIA Project License for Annotated Corpora" here.

**Review by jason-fries — CHANGES_REQUESTED** [review]
Hi @albertvillanova 
This looks pretty good! One small change for the license string. Otherwise ready to merge.

File: biodatasets/genia_term_corpus/genia_term_corpus.py (hunk 1 above)
Comment (albertvillanova) [reply biodatasets/genia_term_corpus/genia_term_corpus.py]: Done @jason-fries.

**Comment by hakunanatasha** [comment]
@jason-fries as the change is addressed, I am merging as it passes all unit tests + has the right behavior.

**Review by hakunanatasha — APPROVED** [review]

**Pull request merged**